dotnet-desktop-guide/net/wpf/documents/snippets/how-to-print-xps-files/vb/MainWindow.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<Window x:Class="CodeSampleVb.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d"
+        Title="Print an XPS file" Height="149" Width="509">
+    <Grid Margin="0,0,0,10">
+        <Label Content="Enter the path to a folder that contains XPS files" HorizontalAlignment="Left" Margin="27,12,0,0" VerticalAlignment="Top" FontSize="11" Height="25" Width="339"/>
+        <TextBox HorizontalAlignment="Left" Margin="32,35,0,0" TextWrapping="Wrap" VerticalAlignment="Top" Width="447" FontSize="14" x:Name="txtFolderPath" Height="21"/>
+        <Button Content="Print" HorizontalAlignment="Left" Margin="32,71,0,0" FontSize="14" Width="51" x:Name="btnPrint" Height="28" VerticalAlignment="Top"/>
```

Comment:
The button click event should be set here

Reply:
Done.